<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,9">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="windowBar" native="true">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#windowBar{
background:gray;
border-top:1px solid gray;
border-right:1px solid gray;
border-left:1px solid gray;
border-top-left-radius:7px;
border-top-right-radius:7px;
}
#windowBar&gt;QPushButton{
border-radius:8px
}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>7</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>7</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
          </font>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>715</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="miniPushButton">
         <property name="minimumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
           <underline>false</underline>
           <stylestrategy>PreferDefault</stylestrategy>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background:#FEE66E;
}
QPushButton::hover{
	background:#c5b355;
}
QPushButton::pressed{
	background:#c5b355;
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../tomato_farm/icon/icons.qrc">
           <normaloff>:/icon/miniShow.png</normaloff>:/icon/miniShow.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closePushButton">
         <property name="minimumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
           <underline>false</underline>
           <stylestrategy>PreferDefault</stylestrategy>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background:#F95D5C;
}
QPushButton::hover{
	background:#aa3e3e;
}
QPushButton::pressed{
	background:#aa3e3e;
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../tomato_farm/icon/icons.qrc">
           <normaloff>:/icon/close.png</normaloff>:/icon/close.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="mainWidget" native="true">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#mainWidget{
background:#e9eaed;
border-bottom:1px solid gray;
border-right:1px solid gray;
border-left:1px solid gray;
border-bottom-left-radius:7px;
border-bottom-right-radius:7px;
}

QTextEdit{
	border:1px solid gray;
	background:white;
    border-radius:10px;
	padding:2px 4px;
}

QLineEdit{
	border:1px solid gray;
    border-radius:10px;
    padding:2px 4px;
}

QComboBox {
	border:1px solid gray;
	background:white;
    border-radius:10px;
	padding:2px 4px;
}

QComboBox::drop-down{
	width: 20px;
	border-left-width: 1px;
	border-left-style: solid;
	border-left-color: gray; 
}

QComboBox::down-arrow{
	border-image:url(:/icon/down.png);
}

QComboBox&gt;QAbstractItemView 
{
	border: 2px solid darkgray;
    selection-background-color: lightgray;
    padding-top:4px;
	padding-bottom:4px;
}
     
QComboBox QAbstractItemView::item
{
    height: 24px;
}
     
QComboBox QAbstractItemView::item:selected
{	
    background-color: rgba(54, 98, 180);
}

QSpinBox,QDoubleSpinBox{
	border:1px solid gray;
	border-radius:10px;
	padding:2px 4px;
}

/*spinbox 抬起样式*/
QDoubleSpinBox::up-button,QSpinBox::up-button {
	subcontrol-origin:border;
    subcontrol-position:right;
    image: url(:/icon/add.png);
    width: 16px;
    height: 20px;       
}
QDoubleSpinBox::down-button,QSpinBox::down-button {
    subcontrol-origin:border;
    subcontrol-position:left;
    border-image: url(:/icon/remove.png);
    width: 16px;
    height: 17px;
}
/*按钮按下样式*/
QDoubleSpinBox::up-button:pressed,QSpinBox::up-button:pressed{
	subcontrol-origin:border;
    subcontrol-position:right;
    image: url(:/icon/add.png);
    width: 16px;
    height: 20px;       
}
  
QDoubleSpinBox::down-button:pressed,QSpinBox::down-button:pressed{
    subcontrol-position:left;
    image: url(:/icon/remove.png);
    width: 16px;
    height: 17px;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,8">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>8</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>8</number>
         </property>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget&gt;QPushButton{
	border:none;
	color:white;
	border-radius:10px;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,7">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QFormLayout" name="formLayout_3">
              <property name="labelAlignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="formAlignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="leftMargin">
               <number>4</number>
              </property>
              <property name="topMargin">
               <number>2</number>
              </property>
              <property name="rightMargin">
               <number>4</number>
              </property>
              <property name="bottomMargin">
               <number>2</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="font">
                 <font>
                  <family>Microsoft YaHei UI</family>
                 </font>
                </property>
                <property name="text">
                 <string>输入设备：</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="device_list">
                <property name="font">
                 <font>
                  <family>Microsoft YaHei UI</family>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QAbstractItemView 
{
	border: 2px solid darkgray;
    selection-background-color: lightgray;
    padding-top:4px;
	padding-bottom:4px;
	border-radius:10px;
}
     
QAbstractItemView::item
{
    height: 26px;
}
     
QAbstractItemView::item:selected
{	
    background-color: rgba(54, 98, 180);
}</string>
                </property>
                <item>
                 <property name="text">
                  <string>1</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>1</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_3">
                <property name="font">
                 <font>
                  <family>Microsoft YaHei UI</family>
                 </font>
                </property>
                <property name="text">
                 <string>输出IP：</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="ip_edit">
                <property name="font">
                 <font>
                  <family>Microsoft YaHei UI</family>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="inputMask">
                 <string>000.000.000.000;0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_4">
                <property name="font">
                 <font>
                  <family>Microsoft YaHei UI</family>
                 </font>
                </property>
                <property name="text">
                 <string>运行端口：</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QPushButton" name="sure_button">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>80</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Microsoft YaHei UI</family>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton{
	background:#6CE872;
}
QPushButton:hover{
	background:#53b357;
}
QPushButton:pressed{
	border:1px solid white;
	background:#53b357;
}</string>
                  </property>
                  <property name="text">
                   <string>启动</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="stop_button">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>80</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Microsoft YaHei UI</family>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton{
	background:#F95D5C;
}
QPushButton:hover{
	background:#aa3e3e;
}
QPushButton:pressed{
	border:1px solid white;
	background:#aa3e3e;
}</string>
                  </property>
                  <property name="text">
                   <string>停止</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="2" column="1">
               <widget class="QSpinBox" name="port_edit">
                <property name="font">
                 <font>
                  <family>Microsoft YaHei UI</family>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>65535</number>
                </property>
                <property name="value">
                 <number>7777</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label">
                <property name="font">
                 <font>
                  <family>Microsoft YaHei UI</family>
                 </font>
                </property>
                <property name="text">
                 <string>运行日志</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTextEdit" name="textEdit">
                <property name="font">
                 <font>
                  <family>Microsoft YaHei UI</family>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="wave_widget" native="true">
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
            </font>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5"/>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../../tomato_farm/icon/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closePushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1058</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>959</x>
     <y>-15</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
